### DISCUSSION TITLE
Request: Can You Guys Add More Kids & UK Channels Like

### DISCUSSION THREAD

**Discussion by Westh977** [iptv-org/community#1108] [Q&A]
![WETA UK Logo](https://github.com/user-attachments/assets/141961b2-7361-4fcf-b004-c9e476590da1)
![R (2)](https://github.com/user-attachments/assets/7e6b6c94-1f34-4490-9789-ee0d1b87ef49)
![dg3cent-27530e74-f9d6-480b-b267-37730763bb4c](https://github.com/user-attachments/assets/e7861bfd-ed75-4c70-86ad-3cf148bdac23)
![dg4p2b2-6628e9aa-88d0-4200-bed4-b656421a2317](https://github.com/user-attachments/assets/6b085eac-0cdc-4968-8f8c-7e6b9613ba81)
![Sprout_HD_Logo_2010](https://github.com/user-attachments/assets/5edc8720-69c6-4af4-839f-18c39ca5ca3b)
Here's My List
WETA UK
Nick Jr. Too UK 
Family Channel
Sprout HD USA East
Universal Kids HD West
Nick 2 USA West
Nicktoons USA West
Nicktoons UK
![Former_%27%27Nick_2%27%27_logo](https://github.com/user-attachments/assets/6dd5da7d-0bd9-4bbf-a339-7ae7e4c337ee)
![nicktoons_2009_logo_green_by_progamechris_dfwwtkx-pre](https://github.com/user-attachments/assets/c26f228c-dab6-4318-912c-ca8881367eca)

Will You Add Those IPTV M3u Playlist & IPTV Service Please Thank You

**Comment by dtngfan85** [comment]
Sprout HD USA East no longer exists, it's Universal Kids

"Nick 2 USA" isn't a seperate channel, it's just what some cable providers call Nickelodeon West.

WETA UK isn't an actual UK channel, it's a subchannel of WETA in Washington DC
"WETA Kids" is just PBS Kids

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
